source/content/guides/drupal-9-from-community.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Create a Drupal 9 Site Using a Community Distribution
+description: Learn how to create a site using a Drupal 9 community distribution.
+cms: "Drupal"
+categories: [get-started]
+tags: [site]
+type: guide
+permalink: docs/guides/drupal-9-from-community
+contributors: [wordsmither]
+date: 07/06/2022
+---
+
+Distributions are pre-made packages that you can use to simplify creating and setting up a Drupal website. Drupal distributions are exceptionally helpful if you want to create a website, but don't want to build it from scratch. 
+
+Start by following the documentation provided with the community distribution; then continue with the following steps.
+
+## Requirements
+
+- You are managing site dependencies using Composer.  If not, you can [convert a standard Drupal site to a Composer managed site](/guides/composer-convert).
```

Comment:
I think people using this guide are just starting the project so I don't think this applies.

The only requirement is having composer installed in your computer?

Reply:
@kporras07 , https://pantheon.io/docs/guides/drupal-9-hosted-md/prepare has the following instructions related to composer, etc - would these apply here instead of what I've got?

Review the documentation on [Git](https://pantheon.io/docs/git), [Composer](https://pantheon.io/docs/guides/composer), and [Terminus](https://pantheon.io/docs/terminus), and install and configure them on your local computer. Pantheon requires Composer 2, at minimum.

Mac users can use [Homebrew](https://brew.sh/) to install Git, Composer, and PHP, along with their required dependencies. Note: Terminus 3 should be used for PHP >= 8.0. Restart the shell or terminal environment after entering the following command:

brew install git composer php
Windows users can install [Composer](https://getcomposer.org/doc/00-intro.md#installation-windows) and [Git](https://git-scm.com/download/win), and may need to install [XAMPP](https://www.apachefriends.org/index.html) or similar to satisfy some dependencies.

Reply:
Yup, that sounds great!